README.md

```diff
@@ -288,7 +288,6 @@ $ citm.printCanMsgs() # Prints all CAN messages
 $ citm.printCanMsgsIso() # prints all CAN messages received on the Isolation network
 ```
 
-Placing a bookmark places a bookmark simultaneously on both the Isolation information and the aggregate information.
+Placing a bookmark places a bookmark simultaneously on both the isolation information and the aggregate information.
```

Comment:
actually, this capital has a purpose for the moment.  let's leave it alone.  we'll talk more about the meanings later, and probably change out the terminology a bit in the future.

Reply:
Ok. Updated.